lib/gui_player_factory.rb

```diff
@@ -0,0 +1,5 @@
+require 'gui_player'
+
+class GuiPlayerFactory 
+
```

Comment:
Don't check in empty stuff that doesn't do anything (even if it is the end of the day)